app/policies/item_request_link_policy.rb

```diff
@@ -12,6 +12,13 @@ def show?
     item.on_order?
   end
 
+  # Check to see if request types include hold or recall.
+  # This is used to make sure live reload can't create a request link for items that aren't holdable/requestable.
```

Comment:
I think we want to be clear this is about hold/recall and not other kinds of requests.

```suggestion
  # This is used to make sure live lookup can't create a request link for items that aren't eligible for hold/recall.
```